sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -328,26 +336,16 @@ public synchronized void onActivityStarted(final @NotNull Activity activity) {
   @SuppressLint("NewApi")
   @Override
   public synchronized void onActivityResumed(final @NotNull Activity activity) {
-    if (!firstActivityResumed) {
-
-      // we only finish the app start if the process is of foregroundImportance
-      if (foregroundImportance) {
-        // sets App start as finished when the very first activity calls onResume
-        AppStartState.getInstance().setAppStartEnd();
-      } else {
-        if (options != null) {
-          options
-              .getLogger()
-              .log(
-                  SentryLevel.DEBUG,
-                  "App Start won't be reported because Process wasn't of foregroundImportance.");
-        }
-      }
+    // finishes app start span
+    @Nullable final SentryDate appStartEndTime = AppStartState.getInstance().getAppStartEndTime();
```

Comment:
L: This block and L216 do pretty much the same, maybe extracting to a method would allow DRY.